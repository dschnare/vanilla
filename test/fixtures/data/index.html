<extends "./page.html">
<data>
{
  type: 'page',
  title: function () {
    return this.dest.split('/').pop().split('.').shift().toUpperCase();
  }
}
</data>
<data summary>
  This is the <strong>summary</strong>.
</data>

<replace body>
  <p>This is the body!</p>
</replace>

<append body>
  <p>This is more content added to the body.</p>
</append>

<replace footer>
  <include "./footer.tpl">{ year: 2013 }</include>
</replace>

<block any>
  this should be ignored.
</block>

<partial widget>
  <div>WIDGET</div>
</partial>