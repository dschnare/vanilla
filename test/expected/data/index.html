<html>
  <head>
    <title>INDEX</title><meta charset="utf-8" />
  </head>
  <body>
    This is the <strong>summary</strong>.
    <p>This is the body!</p><p>This is more content added to the body.</p>
    Copyright 2013
    <div>WIDGET</div>
    
    <div>
      <a href="tmp/data/index.html">INDEX</a>
    </div>
  </body>
</html>